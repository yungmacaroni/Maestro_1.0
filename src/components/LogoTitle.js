import React from "react";
import { View, Text } from "react-native";
import { styles } from "./Styles";

export default function LogoTitle() {
  return (
    <View style={styles.titleBar}>
      <Text style={styles.titleText}>Maestro</Text>
    </View>
  );
}
