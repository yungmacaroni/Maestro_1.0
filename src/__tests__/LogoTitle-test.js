import React from 'react';
import renderer from 'react-test-renderer';
import LogoTitle from '../components/LogoTitle'

test('renders correctly', () => {
  const tree = renderer.create(<LogoTitle />).toJSON();
  expect(tree).toMatchSnapshot();
});