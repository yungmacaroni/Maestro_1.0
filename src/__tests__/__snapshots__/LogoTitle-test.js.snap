// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#1b411f",
      "justifyContent": "center",
      "margin": 0,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "white",
        "fontFamily": "Lemonada_400Regular",
        "fontSize": 30,
        "textAlign": "center",
      }
    }
  >
    Maestro
  </Text>
</View>
`;
